packages/office-ui-fabric-react/src/components/Callout/CalloutPage.tsx

```diff
@@ -60,9 +60,6 @@ export class CalloutPage extends React.Component<IComponentDemoPageProps, any> {
             <p>Real-world examples of this implementation can be seen in administrative interfaces where a particularly difficult-to-understand concept is paired with the iconClassNames.info "i" icon. In this example, Callout - with its tip text - is opened when the user clicks on or hovers over the icon.</p>
           </div>
         }
-        related={
```

Comment:
I was still having trouble with my cache / build in Edge but got them to show up completely removed in Chrome.  Glad to hear a confirmation it worked for you, too!